tests/pos/quoteTest.scala

```diff
@@ -0,0 +1,15 @@
+import scala.meta._
```

Comment:
Is the plan to use this for the public API or only tests? `scala.meta.{Tree,Type,Term,...}` are already defined in https://github.com/scalameta/scalameta/blob/v2.1.3/scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala and get ~50,000 monthly downloads.

Reply:
We could use `scala.meta.quote` instead.

Reply:
> We could use scala.meta.quote instead.

Good idea.

Reply:
Still seems a bit too easy to confuse with scala.meta. How about one of: scala.quote, scala.reify, scala.staging

Reply:
+1 for `scala.staging`

Reply:
How about `scala.quoted`? `staging` and `reify` is both expert-speak.

Reply:
Also, it's a stretch to call macro part of this "staging".